--- webkitgtk-2.50.4/Source/cmake/WebKitMacros.cmake.orig	2025-08-08 11:17:56.304456000 +0200
+++ webkitgtk-2.50.4/Source/cmake/WebKitMacros.cmake	2026-01-23 12:48:45.426052097 +0100
@@ -16,6 +16,8 @@
     # Windows needs a larger bundle size because that helps keep WebCore.lib's size below the 4GB maximum in debug builds.
     if (MSVC AND ${_framework} STREQUAL "WebCore" AND ${_framework}_LIBRARY_TYPE STREQUAL "STATIC")
         list(APPEND gusb_args --max-bundle-size 16)
+    elseif (UNIFIED_BUILD_MAX_BUNDLE_SIZE)
+        list(APPEND gusb_args --max-bundle-size ${UNIFIED_BUILD_MAX_BUNDLE_SIZE})
     endif ()
 
     if (ENABLE_UNIFIED_BUILDS)
